src/Compilers/CSharp/Test/Semantic/Semantics/PatternSwitchTests.cs

```diff
@@ -363,9 +363,12 @@ public static void Main()
 }";
             var compilation = CreateCompilationWithMscorlib45(source, options: TestOptions.DebugExe);
             compilation.VerifyDiagnostics(
-                // (8,18): error CS8121: An expression of type string cannot be handled by a pattern of type int.
+                // (8,18): error CS8121: An expression of type 'string' cannot be handled by a pattern of type 'int'.
                 //             case int i: // error: type mismatch.
-                Diagnostic(ErrorCode.ERR_PatternWrongType, "int").WithArguments("string", "int").WithLocation(8, 18)
+                Diagnostic(ErrorCode.ERR_PatternWrongType, "int").WithArguments("string", "int").WithLocation(8, 18),
+                // (9,17): warning CS0162: Unreachable code detected
+                //                 break;
+                Diagnostic(ErrorCode.WRN_UnreachableCode, "break").WithLocation(9, 17)
```

Comment:
Should we try to avoid cascade here?

Reply:
Yes, good idea...

Reply:
Actually, that is hard to do. If we treat the code as reachable, then we would newly complain, e.g. about fallthrough to another case.